specs/ingestion/common/schemas/source.yml

```
@@ -150,3 +148,36 @@ SourceInput:
     - $ref: '#/SourceCommercetools'
     - $ref: '#/SourceBigCommerce'
     - $ref: '#/SourceJson'
+
+SourceUpdateCommercetools:
+  type: object
+  additionalProperties: false
+  properties:
+    storeKeys:
+      type: array
```

```suggestion
      type: array
      description: Unique and immutable key of the referenced Store
```

description comes from the CT docs directly

---

added and also original code part is updated with descriptions.